docs/CentOS Qcow2 to OVA.md

```diff
@@ -0,0 +1,20 @@
+# Overview
+This guide talks about how to convert the CentOS qcow2 to ova format
+
+# Prerequisite
+- The latest RHEL/CentOS ppc64le machine(virtual/baremetal) with enough diskspace with root access
```

Comment:
We need to export the cloud api key as a pre req.

Reply:
api key is not required for this command, its needed only for the upload and import.